.travis.yml

```diff
@@ -2,20 +2,24 @@ language: java
 addons:
   apt:
     sources:
```

Comment:
Could you reset the spacing how it was initially.  It makes it hard to tell at a glance what changed.  I know the default spacing is weird, but it's the way that it gets set if you run some of the travis cli commands, so I just keep it that way or it keeps flipping back and forth.